### ISSUE TITLE
learning rate and optimizer phase when resuming training 

### ISSUE THREAD

**Issue by YSL0226** [eladrich/pixel2style2pixel#189]
We can resume training by 'checkpoint_path' command.

Did you resume  learning rate and optimizer phase by 'checkpoint_path' command?
Because I didn't find it in the code, maybe I miss it? 
If you did it, could you help me to point it out?
If not, how do we resume learning rate and optimizer phase?
Looking forward to your reply.
